lib/charms/layer/apache_bigtop_base.py

```diff
@@ -19,12 +19,22 @@ class Bigtop(object):
     _roles = set()
     _overrides = {}
 
+    @property
+    def bigtop_base(self):
+        '''Path to the bigtop root directory.'''
+        return self._bigtop_base
+
+    @property
+    def site_yaml(self):
+        '''Path to the site_yaml config file.'''
+        return self._site_yaml
+
```

Comment:
Moved these here to make them override-able in the tests. Also helps with introspection.